.chloggen/mongodb_add_address_res_attr.yaml

```
@@ -0,0 +1,30 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receiver/mongodbreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add `server.address` and `server.port` resource attributes to MongoDB receiver.
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [32810, 32350]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  The new attribute are added to the MongoDB receiver to distinguish metrics coming from different MongoDB instances.
```

```suggestion
  The new resource attributes are added to the MongoDB receiver to distinguish metrics coming from different MongoDB instances.
```
Nit: pluralize `attributes`, and also clarify that they're `resource` attributes.